features_and_tools/firely-server-ingest.rst

```diff
@@ -378,181 +618,133 @@ Known issues
   
   * Please note that FSI will not check or warn you if the database already contains resources of a FHIR version different from that specified via the CLI options ``-f``, ``--fhir-version <R3|R4>`` or ``fhirVersion`` in the config file.
 
-* When importing data from large ``*.ndjson`` files, the memory consumption may be quite high.
 * When importing STU3 resources, the field ``Patient.deceased`` will always be set to ``true`` if it exists. This is caused by an error in the FHIR STU3 specification. In case you would like to use FSI with STU3 resources, please :ref:`contact us<vonk-contact>`.
 * If a resource is present in a workload more than once, the entries may get processed in parallel and a version that is different from the latest may be set as current.
 
 
 Release notes
 -------------
 
-.. note::
-    You can pull the latest version of Firely Server Ingest using the following instructions::
-        
-        dotnet tool update --global Firely.Server.Ingest
-
-
-
-
 Release 5.5.0+
 ^^^^^^^^^^^^^^
 
 The FSI release cycle has been synchronized with the Firely Server release cycle.
 Please refer to the :ref:`Firely Server release notes <vonk_releasenotes>` for the FSI change log.
 
 
-.. _fsi_releasenotes_2.3.0:
-
-
-Release 2.3.0, November 23rd, 2023 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. container:: toggle
```

Comment:
Moving this to a subpage (similar to old FS release notes) would have saved a lot of reformatting, and keep the formatting consistent for each release. Did you consider that?

Reply:
I did consider that. However, then I would need to create an overview page that would reference to this and a separate _release notes_ file. The complexity of this file has not reached the point where that was justified (according to my subjective view)